modules/installation-and-upgrade/pages/container-deployment/actkey-bootstrap-proxy-suma.adoc

```diff
@@ -1,8 +1,16 @@
 == Create an Activation Key for the Proxy
 
+// FIXME: Maybe, we'd better remove this WARNING again, or
+//        add more info.
+
+[WARNING]
+====
+If deploying on {sles}, you can skip some reboots.
+====
```

Comment:
such vogue statements doesn't help user. We should mention on the exact step if that step is applicable on sales or not.